Makefile

```diff
@@ -22,15 +22,33 @@
 
 # Service integration id
 APP_ID = spotify
+# Dependencies
+DEPS = rsvg-convert
 # Default installation destination
 DEST ?= $(HOME)/.local/share/nuvolaplayer3/web_apps
+# Size of PNG app icon
+ICON_SIZE ?= 48
```

Comment:
Change [icon size in Makefile to 128](https://groups.google.com/d/msg/nuvola-player-devel/a-axnC05sf8/a8hEsBolxiIJ).